introduction/contributions.tex

```diff
@@ -25,15 +25,15 @@ \section{Contributions}
 \end{table}
 
 
-The implemented build process ({\cbuild}) is, to our knowledge, the first er ergonomic build process using the {\rust} standard package manager, {\cargo},  for embedded systems.
+The implemented build process ({\cbuild}) is, to our knowledge, the first ergonomic build process using the {\rust} standard package manager, {\cargo},  for embedded systems.
 Other projects have resorted to custom Makefiles to handle the build process and dependencies.
 
 In order to develop the build process ({\cbuild}), {\cargo} had to be modified.
-This resulted in implementing and contributing the a subcommand ({\crustc) to the {\cargo} package manager.
+This resulted in implementing and contributing the subcommand ({\crustc) to the {\cargo} package manager.
```

Comment:
"... the subcommand `({\crustc)` ..."
Dette ser vel egentlig broken ut? Fjerne parantesene og sette inn av sluttende krøll-parantes?